drivers/azure/azureutil/azureutil.go

```diff
@@ -591,13 +611,59 @@ func (a AzureClient) GetAvailabilitySet(resourceGroup, name string) (compute.Ava
 	return a.availabilitySetsClient().Get(resourceGroup, name)
 }
 
-func (a AzureClient) CreateAvailabilitySetIfNotExists(ctx *DeploymentContext, resourceGroup, name, location string) error {
+// CreateAvailabilitySetIfNotExists checks that managed disk option match availability set if it already exists. If the
+// availability set does not already exists than it is created with configured parameters.
+func (a AzureClient) CreateAvailabilitySetIfNotExists(ctx *DeploymentContext, resourceGroup, name, location string, isManaged bool, faultCount int32, updateCount int32) error { // , faultCount int32, updateCount int32) error {
```

Comment:
Purpose of comment `// , faultCount int32, updateCount int32) error {` on this line?